### ISSUE TITLE
Don't let specific imageview receive the zoom inside a ZoomLayout

### ISSUE THREAD

**Issue by lucasrsv** [natario1/ZoomLayout#163]
I'm using the ZoomLayout, inside of it I have some [imageviews that work as a menu](https://user-images.githubusercontent.com/43002509/85646138-cde33280-b671-11ea-86fb-aa0239115acb.png), like that:

```
<layout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools">

    <com.otaliastudios.zoom.ZoomLayout
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:id="@+id/skyZoomLayout"
        app:minZoom="1.0"
        app:maxZoom="5.0"
        app:hasClickableChildren="true">

            <FrameLayout
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:id="@+id/sky">

                    <ImageView
                        android:id="@+id/button1 >

                    <ImageView
                        android:id="@+id/button1 >

            </FrameLayout>

    </com.otaliastudios.zoom.ZoomLayout>

</layout>
```
`
But I don't want these specific imageviews to [receive the zoom](https://user-images.githubusercontent.com/43002509/85645969-7e9d0200-b671-11ea-863c-7009ba438abb.png), because they're a simple static menu on the bottom-left part of the screen. Is there a way to do it?

**Comment by markusressel** [comment]
If you don't want them to be a part of the `ZoomLayout` why did you put them inside of it? Can you put your menu outside of the `ZoomLayout` as an overlay?

**Comment by lucasrsv** [comment]
> If you don't want them to be a part of the `ZoomLayout` why did you put them inside of it? Can you put your menu outside of the `ZoomLayout` as an overlay?

I had tried it, but unsuccesfully, because the `ImageViews` used to disappear. But I just found out the property `android: elevation`, changed it to `2dp` and they're now [appearing without receiving the zoom](https://user-images.githubusercontent.com/43002509/85661507-2ecc3400-b68d-11ea-9c95-ca85eaefdde9.jpg).

Thanks and sorry for the issue, at the end it was a silly problem, but it was my first issue hahah.
